hack/lib/update_olm_manifest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
the `hack/lib/` is used to add libs which are valid for others scripts. E.g [common.sh](https://github.com/operator-framework/operator-sdk/blob/master/hack/lib/common.sh) which is imported in other scripts and has methods which are helpers. 

Otherwise, this script is used to gen/build the OLM manifests to the SDK binary. Am I right? 
So, it should not be in the `/hack/lib/`. Since it is part of the SDK image process, IMO it would be in [hack/image](https://github.com/operator-framework/operator-sdk/tree/master/hack/image).


hack/lib/update_olm_manifest.sh